WBooks/DetailBook/DetailBookViewModel.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  DetailBookViewModel.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 25/08/21.
+//
+
+import Foundation
+import Alamofire
+
+class DetailBookViewModel {
+    
+    let book: Book
+    var comments: [Comment]
+    
+    init(book: Book) {
+        self.book = book
+        self.comments = []
+    }
+    
+    func getComments(action: @escaping ([Comment]) -> () ) {
+        
+        let onSuccess = { (comments: [Comment]) in
+            self.comments = comments
+            action(comments)
+        }
+                
+        let onError = { error in
+            debugPrint(error)
+        }
+        
+        let repository = CommentsRepository()
+        repository.fetchComments(id: book.id, onSuccess: onSuccess, onError: onError)
+    }
+    
+    func getUserComment(id: Int, action: @escaping (User) -> () ) {
+        
+        let onSuccess = { (user: User) in
+            action(user)
+        }
+                
+        let onError = { error in
+            debugPrint(error)
+        }
+        
+        let repository = UserRepository()
+        repository.getUser(id: id, onSuccess: onSuccess, onError: onError)
+    }
+    
+    func validateStatusRent() -> Bool {
```

Comment:
I think the name is not the better. Usually if the return value is a boolean, the func begin with `is`. For example isAvaibleRent

Reply:
Done